internal/uvm/vsmb.go

```diff
@@ -107,6 +108,16 @@ func (uvm *UtilityVM) AddVSMB(ctx context.Context, hostPath string, guestRequest
 			},
 			ResourcePath: vSmbShareResourcePath,
 		}
+
+		if guestRequest != "" {
+			if gr, ok := guestRequest.(guestrequest.GuestRequest); ok {
+				if settings, ok := gr.Settings.(*hcsschema.MappedDirectory); ok {
+					settings.HostPath = share.name
```

Comment:
Can you add a comment here about what share.name and settings.HostPath represent? it's slightly confusing that for the rest of the function the parameter "hostPath" is the server host path, not the container host path, whereas settings.HostPath is the latter.

Reply:
yep!